sys/include/net/gnrc.h

```diff
@@ -144,9 +144,10 @@
  * }
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * @note It is necessary to react with a reply message to the other thread for
- *       @ref GNRC_NETAPI_MSG_TYPE_SET and @ref GNRC_NETAPI_MSG_TYPE_GET
- *       commands
+ * @note When receiving a message of type @ref GNRC_NETAPI_MSG_TYPE_SET or
+ *       @ref GNRC_NETAPI_MSG_TYPE_GET, it is necessary to acknowledge it by
+ *       calling @ref msg_reply() with a message of type
+ *       @ref GNRC_NETAPI_MSG_TYPE_ACK.
```

Comment:
allright, but what exactly is "the returned value"? the value returned by the function that was called in reaction to the receipt of the reply-triggering message?

Also, I'm wondering if it makes sense to additionally document this somewhere more prominent than that tutorial ^^